src/common/components/game/Timer.js

```diff
@@ -105,13 +104,13 @@ export default class Timer extends Component {
   }
 
   render() {
-    if (this.props.isTutorial) {
+    if (!this.props.enabled) {
       return null;
     } else {
       return (
         <div style={this.styles.timer}>
           <span style={this.state.timerStyle}>
-            {this.props.isTutorial ? '' : this.state.timer}
+            {this.state.timer}
```

Comment:
Before this would show for game over and practice mode, this is a change in behavior. If that is desired, that's fine. This might also be something I would expect to trigger a test, but since it's just a UI of showing the timer and not some critical business logic, I'm fine approving without a test.